src/web/mod.rs

```diff
@@ -759,7 +763,7 @@ mod test {
                 .create()
                 .unwrap();
             let web = env.frontend();
-            assert_redirect("/bat//", "/bat/latest/bat/", web)?;
+            assert_redirect_unchecked("/bat//", "/bat/", web)?;
```

Comment:
This redirect is handled by `iron`, so there's no easy way to make it go directly to the final target.